packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -556,6 +563,8 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
   }
 
   private _forceListUpdates() {
+    this._pendingForceUpdate = false;
+
     if (this._groupedList) {
```

Comment:
Part of me feels like we should make forceUpdate a property that gets passed down to the grouped/list so that it maintains the lifecycle better. Probably doesn't need to be done now.

Reply:
We should get *rid* of `forceUpdate`.